src/main/java/seedu/address/logic/parser/LoginCommandParser.java

```diff
@@ -3,6 +3,8 @@
 import seedu.address.logic.commands.LoginCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
 
+//@@author: IcedCoffeeBoy
```

Comment:
RepoSense is used for grading, as Collate is not used anymore. Not sure whether this should be placed, as Jeevan will be adding some stuff to it in his latest PR.